packages/core/src/components/Chart/Chart.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import * as Classes from '../../common/classes'
+
+const CHART_URL = 'https://bitfinexcom.github.io/bfx-hf-tradingview'
+
+const DEFAULT_THEME = 'default-theme:light-mode'
+
+const Chart = ({
+  market: {
+    wsID,
+    base,
+    quote,
+  },
+  theme,
+}) => {
+  const queryString = new URLSearchParams({
+    wsID,
+    base,
+    quote,
+    apiBaseUrl: 'https://api-pub.bitfinex.com',
```

Comment:
please use apiBaseUrl from config.selectors by defining new param:
`export const pubApiUrl = process.env.REACT_APP_UFX_PUBLIC_API_URL || 'https://api-pub.bitfinex.com'`

It will however make `apiUrl/pubApiUrl` both use same env REACT_APP_UFX_PUBLIC_API_URL.
I will make them use two separate envs `REACT_APP_UFX_API_URL/REACT_APP_UFX_PUBLIC_API_URL` in separate PR